src/core/meshcop/commissioner.hpp

```diff
@@ -440,8 +440,8 @@ class Commissioner : public InstanceLocator, private NonCopyable
                                               Error                aResult);
     void HandleLeaderKeepAliveResponse(Coap::Message *aMessage, const Ip6::MessageInfo *aMessageInfo, Error aResult);
 
-    static void HandleSecureAgentConnected(bool aConnected, void *aContext);
-    void        HandleSecureAgentConnected(bool aConnected);
+    static void HandleSecureAgentConnected(otCoapSecureConnectEvent aEvent, void *aContext);
```

Comment:
define `typedef otCoapSecureConnectEvent CoapSecureConnectEvent;` in coap_secure.hpp and use `CoapSecureConnectEvent` type in core?

Reply:
I think I missed to update this to `MeshCoP::SecureTransport::ConnectEvent`, updated.